docs/swagger.yaml

```diff
@@ -381,6 +381,75 @@ paths:
                       message:
                           type: string
                           example: 'user not found'
+  /novels/{commentId}/like:
+    post:
+      summary: Creates a comment
+      description: Post a comment on a novel
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                commentBody:
```

Comment:
Good job, but I don't think comment body should be part of this parameter

Reply:
ill fix this now

Reply:
done